src/platform/packages/shared/kbn-dom-drag-drop/src/sass/droppable.scss

```diff
@@ -51,14 +51,14 @@
 }
 
 .domDroppable--incompatible:not(.domDroppable__overlayWrapper) {
-  background-color: $euiColorHighlight !important;
+  background-color:  transparentize($euiColorWarning, .9) !important;
   &:before {
-    border: $euiBorderWidthThin dashed $euiColorVis5 !important;
+    border: $euiBorderWidthThin dashed $euiColorWarning !important;
```

Comment:
I believe this border color variable should be changed to `$euiColorBorderBaseWarning`. Please note that this variable may be bugged/erroneous at the moment, but the EUI team is working on a PR to remedy it.

Reply:
done in [49d8096](https://github.com/elastic/kibana/pull/204839/commits/49d8096ab969b7f4b1a834b4ebba60b373c80808)